project_one/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Please pass me</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<span>This is section one</span> 
+	<button id="grading-element-one" type="button">Add A Thing</button>
+	<ul id="grading-element-one-consequence"></ul>
+	<br>
+	<a id= "grading-element-two" href = "about.html">About</a> <br>
+	<span>This is section three</span>
+	<button id="grading-element-three" type="button">Show A Thing</button> <br>
+	<span id="grading-element-three-consequence"></span>
```

Comment:
close...but this should start with text already in it, and `visibility: hidden`. Then in the function that fires on the `grading-element-three` click event, turn the `visibility` to `visibility: visible`.